lesson7/directions.py

```diff
@@ -0,0 +1,27 @@
+
+
+def directions(string: str) -> int:
+    list_dir = []
+    count_up = 0
+    count_down = 0
+    count_left = 0
+    count_right = 0
+    for part in string_dir:
+        list_dir.append(part)
+    for element in list_dir:
+        if element == '^':
+            count_up +=1
+        elif element == 'v':
+            count_down +=1
+        elif element == '<':
+            count_left +=1
+        else:
+            count_right +=1
```

Comment:
lines from 5 to 19 can be replaced with just 4 lines. just remember that python already have lots of solutions inside